controller.rst

```diff
@@ -241,10 +251,9 @@ are used for rendering templates, sending emails, querying the database and
 any other "work" you can think of. When you install a new bundle, it probably
 brings in even *more* services.
 
-When extending the base controller class, you can access any Symfony service
+When extending the base ``Controller`` class, you can access any Symfony service
```

Comment:
Hmm it seems I got confused, reading @nicolas-grekas's comment above:
>In fact, there is no config to do at all, an AbstractController doesn't need to be a service at all
The injection in this case is relying on an argument value resolver, so the controller does not need any configuration indeed.